chat-widget/src/components/livechatwidget/common/defaultProps/dummyDefaultProps.ts

```diff
@@ -1459,7 +1459,8 @@ export const dummyDefaultProps: ILiveChatWidgetProps = {
             // "bundle" props: these stuff are not available if you "decompose" or "recompose" Web Chat.
             //                 I.e. not using composition mode (or <Composer>).
             className: undefined,
-            role: undefined
+            role: undefined,
+            receivedMessageAnchorColor: "white"
```

Comment:
webChatProps are the formal interface expoesd by WebChat to be consumed in <WebChat> React component. Please do not modify this interface.

Reply:
Let's add receivedMessageAnchorColor to IRenderingMiddlewareProps